plugins/meson.build

```diff
@@ -13,9 +13,12 @@ subdir('nitrokey')
 subdir('pci-bcr')
 subdir('pci-mei')
 subdir('test')
-subdir('upower')
 subdir('pixart-rf')
 
+if not get_option('bsd')
```

Comment:
assuming you successfully detect at build time, it should be guarded behind that flag rather than an option

Reply:
And if you can't detect from meson that it's a linux build, we should instead offer a meson option for this plugin rather than one for "bsd"

Reply:
According to docs https://mesonbuild.com/Cross-compilation.html you can use `host_machine.system()`

Reply:
I've pulled out some of the stuff related to this to https://github.com/fwupd/fwupd/pull/2896 while I was looking at it.

Reply:
@superm1 I'll change this in the close future. I was trying to build fwupd on DragonflyBSD today and I've pushed WIP commit to the upstream PR by mistake. I will fix following problems in the next push force.

Reply:
Something like what I did landed on master.  So please rebase all your work on master, and you shouldn't need to have a `get_option('bsd')` anymore.